splice/queries/distribution.py

```diff
@@ -154,3 +164,48 @@ def get_possible_distributions(today=None, channel_id=None):
             "force_upload": True
         })
     return artifacts
+
+
+_SENTINEL = object()
+
+
+def multiplex_directory_tiles(tiles):
+    """Directory tile multiplexer that creates all possible combinations of the
+    tile sets based on its type and the target url. Given multiple sponsored tiles,
+    it'll pick one of them and create a new tile set with other directory tiles.
+    It also multiplexes the tiles with the same TLD+1 target url (e.g. www.mozilla.org).
```

Comment:
So, for example, this wouldn't be possible anymore:

<img src="http://f.cl.ly/items/0B1w3h1H2j1Y2u2T442m/Screen%20Shot%202015-11-17%20at%2010.12.53%20AM.png" />